content/community/newsletter-archive.md

```diff
@@ -7,6 +7,7 @@ online: false
 
 2022
 
+- [February 2022](https://newsletters.getty.edu/t/t-7029ED74D2C80EB52540EF23F30FEDED): Introducing a Sample Quire Workflow, Pandoc: Converting Word files into Markdown, New & Uploaded Documentation, Museum Publishing Digital Interest Group, Philadelphia Museum of Art, Mills College Museum of Art
```

Comment:
Thansk @dimrkh. Quick note. It should be "New and **Updated** Documentation."

Reply:
@Erin-Cecele fixed it; thanks for catching that!!